### ISSUE TITLE
Provide Slightly Extended variant as well

### ISSUE THREAD

**Issue by N-R-K** [shytikov/pragmasevka#5]
Hi,

Very nice project. I almost couldn't tell the difference between it and PragmtaPro (the `&` gives it away).

However, one major problem I have with PragmataPro (and Iosevka) is that I cannot look at them for extended period of time due to how condensed they are - it gives me eye strain. The "Extended" variant of Iosevka mostly fixes this issue for me.

It would be very nice if this project can provide the extended variant as well - although I understand that an extended version would go against the project's goal of mimicking PragmataPro.

Thanks.

**Comment by N-R-K** [comment]
Decided to give the iosevka builder a try myself with the config in this repo. Turns out a width of `550` is ideal for me, it's just extended enough to not strain my eyes (but not as extended as Iosevka's default "Extended" profile, which uses 600).

Of course, with the immense amount of customization of Iosevka at hand, I ended up changing a bunch of *other* things outside just the width. Here's the result: https://github.com/N-R-K/Iosvmata. Not exactly "Pragmasevka Extended" like this issue asks for, but maybe someone else finds it useful too.

- - -

I'll still keep this issue open, as I think a "Slight Extended" variant (550 width) would be a great addition to this project.
